### PR TITLE
Use Summary for posts in landing page

### PR DESCRIPTION
**Pull request by ananyo2012**
- Use pelican summary plugin for summaries

- Failed attempt to add pagination due Bug in pagination - getpelican/pelican#1223

### CODE REVIEW

**Comment by ananyo2012** [comment]
Check the netlify link for preview changes. The css is broken though since the site is not deployed in `/blog` url. I tried to check if its possible in netlify but couldn't find an option.

**Comment by ananyo2012** [comment]
> We had some people working on this issue at PyDelhi DevSprints and he came across the same bug as well. I think we can leave pagination for now.

We can try by totally disabling categories. I think we don't use them anyway.

> Where to check the demo link? I am not sure I have used Netlify.

Click the `Show all checks` button. It will have a `deploy/netlify` option where the `Details` link contains the preview link.

**Comment by vipulgupta2048** [comment]
I checked it out through the link, the implementation looks solid. Should we deploy it and see how it looks overall?

**Comment by ananyo2012** [comment]
Here is the screenshot with the proper css
screenshot from 2018-06-30 12-59-48

I have also added pagination by disabling the categories.

File: themes/inpycon2018/templates/pagination.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+<p class="paginator">
+    {% if articles_page.has_previous() %}
+        <a href="{{ SITEURL }}/{{ articles_previous_page.url }}">&laquo;</a>
+    {% endif %}
+    Page {{ articles_page.number }} / {{ articles_paginator.num_pages }}
+    {% if articles_page.has_next() %}
+        <a href="{{ SITEURL }}/{{ articles_next_page.url }}">&raquo;</a>
+    {% endif %}
+</p>
+{% endif %}
```
Comment (realslimshanky) [inline themes/inpycon2018/templates/pagination.html]: Please add a newline here.

Comment (ananyo2012) [reply themes/inpycon2018/templates/pagination.html]: Fixed

**Comment by ananyo2012** [comment]
Merging this.

**Pull request merged**